README.md

```diff
@@ -1 +1,192 @@
-# stock-exchange_metrics-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="https://raw.githubusercontent.com/microverseinc/readme-template/master/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Metrics-Web-App <a name="about-project"></a>
+
+**Metrics-Web-App** is a mobile web application to that is used to check a list of metrics (numeric values) that will be created by making use of React and Redux and Data from the ACNH API.
+
+<div align="center">
+  <a href="https://ibb.co/DbFM7dv"><img src="https://i.ibb.co/MhH1Rdt/3-devices-white.png" alt="3-devices-white" border="0"></a>
+  <br/>
+</div>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+    <li><a href="">CSS</a></li>
+    <li><a href="https://html.com/">HTML</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Description of 1-3 key features of the application.
+
+- **When the page loads, the webapp shows the list of art pieces that could be filtered out by some parameter**
+- **These art pieces are gotten from an API and when the user clicks (or taps) on a piece, the application navigates to the details page.**
+- **On the details page, the webapp retrieves data from the API to show detailed data of the piece that was clicked on and when the user clicks on the "Back" button (<), the user navigates back to the home page**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
```

Comment:
- [ ] it would be better if you add deployed version link as required in order to complete the requirements 
- Screenshot of the requirements 
<img width="1105" alt="Screenshot 2022-12-01 at 12 07 07" src="https://user-images.githubusercontent.com/61790587/205037415-a6eeee33-9ec4-42f9-aee9-2b44b6a583aa.png">

Reply:
remove spacing

Reply:
- Please deploy your project using Heroku, Netlify, or GitHub Pages.
- Make sure the project is deployed properly, and the home and detail page works as expected.
- The link to the deployed project is present in the README.